src/table/TBody.tsx

```diff
@@ -182,13 +182,13 @@ export default function TBody(props: TableBodyProps) {
 
   // 垫上隐藏的 tr 元素高度
   const translate = `translateY(${virtualConfig.translateY}px)`;
-  const posStyle = virtualConfig.isVirtualScroll
-    ? {
+  const posStyle: React.CSSProperties = virtualConfig.isVirtualScroll
```

Comment:
```suggestion
  const posStyle: React.CSSProperties | undefined = virtualConfig.isVirtualScroll
```
是不是需要加上这个，`React.CSSProperties` 应该是没有 optional 的吧